src/broken_spec/views/search_criteria/SearchCriteriaView_spec.js

```diff
@@ -48,7 +48,7 @@ describe.skip('Search Criteria View', function () {
             }
         };
 
-        resultsCollection = new SearchResultsCollection({model: searchParameters});
+        resultsCollection = new SearchResultsCollection(null, {model: searchParameters});
```

Comment:
Even though some of these specs are in the "broken" category right now, I updated the function call so there would be one less thing to confuse our future selves.